CONTRIBUTING.md

```diff
@@ -120,7 +120,7 @@ If you're not able to use the dev container, follow these instructions:
 
 1. Make sure [Rust](https://www.rust-lang.org/tools/install) is installed.
 2. Install [wasm-pack](https://rustwasm.github.io/wasm-pack/installer/) with `cargo install wasm-pack` .
-3. Node version >= `16.8.0`.
+3. Node version >= `18`.
```

Comment:
The main package.json still allows v16.8

Reply:
I changed that section because that was the one I was using, following step by step, when running `pnpm install`, it threw an error saying I needed 18.
I installed `pnpm` with `brew install pnpm`, so I guess it installs the latest. If we need a specific version of `pnpm` that actually supports `Node version >= 16.8.0`, I guess it would be a good idea to specify it as well